.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: Node CI
```

Maybe `Build`?
Also, please rename the file.
```suggestion
name: Build
```

---

Fixed